design/index.md

```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
 
 Blender is a free and open-source 3D computer graphics software toolset. It can be used to create static renders of 3d assets, or entire 2d & 3d animations. 
 
-### Websites
+## Websites
 
 - All written content should be in UK English.
 - All page titles should summarise the content, keep the URL length as short as practical and use [Kebab Case](https://en.wiktionary.org/wiki/kebab_case).
 - All images should use informative [alt tags](https://www.w3.org/WAI/tutorials/images/tips/) which clearly describe the point of an image rather than all the details. Alt tags should be no longer than 60 characters.
 - When mentioning [FlowForge Concepts](https://flowforge.com/docs/user/concepts/) (terminology) where possible we should link to an explanation of that concept.
+
+## Social Media, Marketing & PR
+
+IT is important we understand what we communicate, to who, how and when. This sets out our principles for social media, marketing and PR communications.
+
+### Content Types
+
+Content should be published to the appropriate channels, the content we create is as follows.
+
+#### Release
+
+Major releases of FlowForge, the timing of these releases are governed by our [development cadence](https://flowforge.com/handbook/development/#cadence).
+
+#### Patch
+
+Patch between releases, these will happen as needed and may be at short notice.
+
+#### Newsletter
+
+Monthly newsletter published at the start of each calendar month covering news from the prior month.
+
+#### How To
+
+Technical guides, published once per week.
+
+#### Planned Downtime
+
+As soon as we are aware of planned downtime we should publish the information, that should be at least one week before the downtime.
+
+#### Outage
+
+If we experience unplanned downtime we should communicate in near real time to keep our users and stakeholders up to date with the situation.
+
+#### Corporate Comms
+
+Major company news such as investment rounds should be communicated at the time agreed with the relevant parties involved, it is important there is no significant delay in communicating this information.
+
+### Content Channels
+
+Each content type has channels which are appropriate, this table sets out which content should be posted on each channel.
+
+|Content          |Blog  |Mailing List |Twitter  |Youtube  |LinkedIn |Reddit |Node-RED Slack|
+|---              |---   |---          |---      |---      |---      |---    |---           |
+|Release          |*     |             |*        |*        |*        |*      |*             |  
+|Patch            |      |             |*        |         |         |*      |*             |   
+|Newsletter       |*     |*            |*        |         |*        |*      |*             |
+|How To           |      |*            |*        |         |*        | *     |*             |  
+|Planned Downtime |      |*            |*        |         |*        |*      |*             |  
```

Comment:
```suggestion
|Planned Downtime | *     |?            |*        |         |*        |*      |*             |  
```

I think our mailing list is the highlights of the past 30 days or so. Planned downtime might occur in before a newsletter goes out too?

Or is the Mailing list not the same as the monthly newsletter to the mailing list?

Reply:
Sorry, this was an error in my MD table, I don't think we should send emails about planned downtime.

Reply:
Hopefully resolved in https://github.com/flowforge/handbook/pull/133/commits/a01d7df2db115203663ec7ed363ee0049aecdb30